letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -912,6 +912,34 @@ OldVenvExists() {
     [ -n "$OLD_VENV_PATH" -a -f "$OLD_VENV_PATH/bin/letsencrypt" ]
 }
 
+# Given python path, version 1 and version 2, check if version 1 is outdated compared to version 2.
+# Unofficial versions (eg. 0.28.0.dev0) will be treated specifically by printing "UNOFFICIAL".
+# Otherwise, print "OUTDATED" if version 1 is outdated, or "UP_TO_DATE" if not.
+# This function relies only on installed python environment (2.x or 3.x) by certbot-auto.
+CompareVersions() {
+    "$1" - "$2" "$3" << "UNLIKELY_EOF"
+import sys
+from distutils.version import StrictVersion
+
+try:
+    current = StrictVersion(sys.argv[1])
+except:
+    sys.stdout.write('UNOFFICIAL')
```

Comment:
It's idiomatic to say merely `print('UNOFFICIAL')`, unless you have a reason not to. If you're dodging the parenthesis requirement of Python 3, instead `from __future__ import print_function` at the top.

Reply:
I am doing it intentionally, to avoid an endline that I would need to strip after when output of this function is assigned to the version variable.